bool door_open_msg
